xformers/triton/patch.py

```diff
@@ -0,0 +1,41 @@
+import typing
+
+import torch
+import torch.nn
+import torch.nn.functional
+
+from dropout import dropout
```

Comment:
Should these be absolute, e.g., `from xformers.triton.dropout import dropout`?

Reply:
Good catch! I'll change that immediately. Thank you!